### DISCUSSION TITLE
Running this example results in an error-"Modeling: Sweep chain and sweep sketch"

### DISCUSSION THREAD

**Discussion by xxtemps** [ansys/pyansys-geometry#1141] [Q&A] [answered]
![1713450684244](https://github.com/ansys/pyansys-geometry/assets/12379914/415cd3e6-9984-42d2-91dc-64fb34bcded6)
What is the cause of this error? thank you

**Comment by RobPasMue** [comment]
Looks like a vtk/trame issue. Could you let me know what versions of vtk and trame are you using? Easiest way is that you copy paste the output of this command:

```
pip freeze
```

**Reply by xxtemps** [reply]
![image](https://github.com/ansys/pyansys-geometry/assets/12379914/df61dea6-8dd9-4a0d-828d-35658d71767b)
![image](https://github.com/ansys/pyansys-geometry/assets/12379914/030fb090-eafd-43b5-815a-9e8de9971593)

**Reply by RobPasMue** [reply, marked as answer]
For some reason your dependencies seem to be quite old. I believe that you might be using an old virtual environment. My recommendations are:

Option 1: create a new virtual environment and install ``ansys-geometry-core`` again from scratch
Option 1bis: what you can also do is install the tests target of ansys-geometry-core
```
pip install ansys-geometry-core[tests]
```
This will make you have the same pinned dependencies we have when testing.

Option 2: force the upgrade of the VTK and trame related packages:
```
pip install --upgrade --force-reinstall 'trame>=2.5.2' 'trame-client>=2.12.7' 'trame-server>=2.11.7' 'trame-vtk>=2.5.8' 'trame-vuetify>=2.3.1'
```

**Comment by xxtemps** [comment]
![image](https://github.com/ansys/pyansys-geometry/assets/12379914/486bc0c5-32ae-4e52-8a71-41c6d7b0e0fb)

The above problem is solved, and there are new problems. 😂Is it also because the dependencies are too old?

**Reply by RobPasMue** [reply]
Awesome, at least the old problems are solved! And no, the new problem is not due to dependencies 😆 

Now, this new problem is caused because the features you are trying to use are only available in Ansys 24R2. Unless you have access to the pre-release version you won't be able to perform a ``sweep_sketch()`` operation. We might need to improve the backend version message a bit more (sorry for that!)

**Reply by RobPasMue** [reply]
I just opened an issue to improve the message - see #1143

**Reply by wangleitheone** [reply]
When will the 24R2 be released, please? I'm interested in PyAnsys and hope to use a more full-featured version soon.

**Reply by RobPasMue** [reply]
Hi @wangleitheone! I might have to verify this date, but this should be coming in the next few weeks (mid May - early June). Stay tuned! I'll keep you posted.
